walkthroughs/week-3-server/examples/random-quotes/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
You have a lot of changes to the walkthroughs in this PR.  It's fine to modify them, but don't those changes into git

Reply:
I stopped checking them in after part 2 but I also notice that those files as well as the commits from the last part keep showing up, I think it has something to do with the fact that this branch and the last branch aren't merging anymore and I have to do it manually I think something went wrong

Reply:
I meant undo the changes, not delete the files.  You can use "git revert" to revert any changes before you commit.

Reply:
I thought git rm -r <file> just removed files from git not from the computer so that's why some got deleted. I added them back but I'm trying resolve the automatic merging problem and it's saying that several of my local and public branches have diverged, but I'll use git revert in the future.

Reply:
git rm does remove the files.  The issue here is that you don't want to remove them, just revert them to an older state